client/src/lib/dependency-injection/applyFormMiddleware.js

```diff
@@ -1,7 +1,6 @@
 import Injector from '../Injector';
 import getIn from 'redux-form/lib/structure/plain/getIn';
 import setIn from 'redux-form/lib/structure/plain/setIn';
-import SchemaStateManager from './SchemaStateManager';
 
 const applyFormMiddleware = (reducer) => () => (state, action) => {
```

Comment:
I do wonder if we should exclude executing this when `action.type == '@@redux-form/REGISTER_FIELD'`, as it's a fair amount of overhead for little gain for this to be firing every redux-form action or at least for registering a field.